library/psa_crypto_driver_wrappers.h

```diff
@@ -328,6 +328,22 @@ psa_status_t psa_driver_wrapper_mac_verify_finish(
 psa_status_t psa_driver_wrapper_mac_abort(
     psa_mac_operation_t *operation );
 
+
+/*
+ * Entropy entry points
+ */
+#define PSA_DRIVER_GET_ENTROPY_BLOCK  1
```

Comment:
This belongs in a header in `include/psa`. This was missing from the specification (my bad). I've created one in https://github.com/ARMmbed/mbedtls/pull/5464. (Work in progress, please don't rebase on top of it yet.)

Reply:
OK. I've moved it into a separate commit to remove it easily when needed.